package.json

```
@@ -0,0 +1,89 @@
+{
+	"name": "@skyra/iris",
+	"version": "1.0.0",
+	"description": "Suggestion management bot for Discord",
+	"author": "Skyra Project <contact@skyra.pw>",
+	"license": "MIT",
+	"private": true,
+	"main": "dist/main.js",
+	"type": "module",
+	"imports": {
+		"#lib/*": "./dist/lib/*.js",
+		"#lib/structures": "./dist/lib/structures/index.js"
+	},
+	"scripts": {
+		"build": "tsc -b src",
+		"dev": "yarn build && yarn start",
+		"watch": "tsc -b src -w",
+		"watch:start": "tsc-watch -b src --onSuccess \"yarn start\"",
+		"clean": "node scripts/build/clean.mjs",
+		"start": "node --enable-source-maps dist/main.js",
+		"test": "eslint --ext ts src",
+		"lint": "eslint --fix --ext ts src",
+		"format": "prettier --write --loglevel=warn \"src/**/*.{js,ts,json}\"",
+		"update": "yarn upgrade-interactive",
+		"prepare": "husky install .github/husky"
```

```suggestion
		"postinstall": "husky install .github/husky"
```

prepare no worky for yarn v3 :( 
(yes other repos also need changing, feel free to do it if you feel like it. For bots postinstall works, for libraries see i.e. shapeshift where it uses pinst)